[{"user_id": 1398, "stars": [], "topic_id": 8206, "date_created": 1298423945.526412, "message": "Compare http://www.djangoproject.com/community/packages/ with the latest 5 apps on http://www.djangopackages.com. No comparison. I've submitted the DP latest packages feed but it is not getting picked up. I've reported this in IRC, email and twitter. Should I open a ticket?", "group_id": 81, "id": 185361}, {"user_id": 15046, "stars": [], "topic_id": 8206, "date_created": 1298433735.8653021, "message": "+1, this would be great.  http://www.djangoproject.com/community/packages/ makes it look as if Django hardly has any regularly-updated packages.", "group_id": 81, "id": 185690}, {"user_id": 1822, "stars": [], "topic_id": 8206, "date_created": 1298435060.548068, "message": "judging from this - you just find a person with admin access to the site and poke them hard enough to get the feed added: http://code.djangoproject.com/browser/djangoproject.com/django_website/apps/aggregator", "group_id": 81, "id": 185721}, {"user_id": 6688, "stars": [], "topic_id": 8206, "date_created": 1298439990.562777, "message": "@pydanny Not sure what's going on there -- I can confirm that http://djangopackages.com/feeds/packages/latest/atom/ is in our list of feeds", "group_id": 81, "id": 185985}, {"user_id": 6688, "stars": [], "topic_id": 8206, "date_created": 1298440045.379225, "message": "@pydanny Best guess would be that there is a cron job that hasn't run yet", "group_id": 81, "id": 185986}, {"user_id": 6688, "stars": [], "topic_id": 8206, "date_created": 1298440090.7171841, "message": "The people to bug will be @justinlilly and @jezdez -- Justin wrote the community pages, and Jannis has root on the web box.", "group_id": 81, "id": 185988}, {"user_id": 213, "stars": [], "topic_id": 8206, "date_created": 1298456443.110944, "message": "@pydanny Opening a ticket would have been the best way, thanks.", "group_id": 81, "id": 186544}, {"user_id": 213, "stars": [], "topic_id": 8206, "date_created": 1298457443.3862081, "message": "Fixed now. Again, please open a ticket next time, we can't roam all online resources for potential bugs all the time.", "group_id": 81, "id": 186621}, {"user_id": 1398, "stars": [], "topic_id": 8206, "date_created": 1298466513.565084, "message": "alright - wasn't sure since the ticket categories was unclear.", "group_id": 81, "id": 187389}, {"user_id": 6396, "stars": [], "topic_id": 8206, "date_created": 1298484784.924685, "message": "@jezdez To be fair, he came here asking whether to open a ticket - not to file a bug report.", "group_id": 81, "id": 189419}, {"user_id": 213, "stars": [], "topic_id": 8206, "date_created": 1298506128.16868, "message": "@pydanny there is a ticket component called \"Django Web site\", that's usually what fits best", "group_id": 81, "id": 193671}]